manifests/profiles/openshift.yaml

```diff
@@ -17,6 +17,6 @@ spec:
       logLevel: info
       privileged: true
       provider: "multus"
-    sidecarInjectorWebhook:
-      injectedAnnotations:
-        k8s.v1.cni.cncf.io/networks: default/istio-cni
+    istio_cni:
+      enabled: true
```

Comment:
We don't need `enabled`, but we needed `chained` due to how the operator code merges in differences between the CNI chart and the istio-discovery chart.

We can remove `enabled` if it would make it more clear.